AndroidSDK/src/com/leanplum/LeanplumPushService.java

```diff
@@ -320,12 +318,34 @@ private static void showNotification(Context context, Bundle message) {
     if (message.getString("title") != null) {
       title = message.getString("title");
     }
-    NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
-        .setSmallIcon(context.getApplicationInfo().icon)
+    NotificationCompat.Builder builder;
+
+    // If we are targeting API 26, try to find supplied channel to post notification.
+    if (LeanplumNotificationChannel.isNotificationChannelSupported(context)) {
+      String channel = message.getString("lp_channel");
```

Comment:
please put here try/catch block with handleException and null checker for builder at 344